homeassistant/components/econet/sensor.py

```diff
@@ -13,9 +13,9 @@
 ENERGY_KILO_BRITISH_THERMAL_UNIT = "kBtu"
 
 TANK_HEALTH = "tank_health"
-AVAILIBLE_HOT_WATER = "availible_hot_water"
+AVAILABLE_HOT_WATER = "available_hot_water"
```

Comment:
This changes code, have you been able to test this integration?

Reply:
No. This code was introduced in #44427 by @w1ll1am23, perhaps they could check?

Reply:
Yeah... This just looks like I spelled it wrong which makes sense because I'm terrible at spelling. 

Looks like this would be a breaking change though since sensor names are based off of it.

Reply:
Can you help me write a Breaking change entry for this?

Reply:
So sorry its been crazy around here and just noticed this email in my inbox. 

The breaking change you have listed looks good to me.